src/mongocxx/uri.hpp

```diff
@@ -170,6 +181,168 @@ class MONGOCXX_API uri {
     /// @return A write_concern that represents what was specified in the uri.
     ///
     class write_concern write_concern() const;
+    ///
```

Comment:
style nit, newline above this comment block

Reply:
Done.